editoast/src/schema/railjson.rs

```diff
@@ -0,0 +1,32 @@
+
+#[derive(Deserialize, Derivative, Serialize, Clone, Debug)]
+#[derivative(Default)]
+#[serde(deny_unknown_fields)]
+pub struct RailJson {
```

Comment:
This is a duplicate from `schema/mod.rs`

Reply:
right I forgot removal